jsonthebuilder/src/components/ResumeField.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import myField from "./ResumeField.scss";
+
+export default function ResumeField(props) {
+
+  
+
+  return (
+    <li className={myField}>
+      {props.name}{<br></br>}
+      {props.label}{<br></br>}
+      {props.email}{<br></br>}
+      {props.website}{<br></br>}
+      {props.location.city}, {props.location.region}{<br></br>}
+      {props.work}
+    </li>
+  )
+}
```

Comment:
Prefer doing destructuring over function parameters and adding some React proptypes.
And use a p tag to do line jumps over the `<br />`

```suggestion
function ResumeField({ name, label, email, website, location, work }) {
  return (
    <li className={myField}>
    <p>{name}</p>
    <p>{label}</p>
    <p>{email}</p>
    <p>{website}</p>
    <p>{location.city}, {location.region}</p>
    <p>{work}</p>
    </li>
  )
}

ResumeField.propTypes = {
  name: PropTypes.string.isRequired,
  label: PropTypes.string.isRequired,
  // and so on...
}

export default ResumeField
```